rfd/xxxx-config-resource-docs.md

```diff
@@ -552,3 +560,9 @@ table above the example YAML) to see an example YAML document for this type.
 If the type of the field is an embedded struct, the generator will act as if the
 fields of the embedded struct are fields within the outer struct. Otherwise, the
 generator will follow the rules above.
+
+## Test plan
+
+We can add an item to the "Documentation" section of the test plan ensuring that
```

Comment:
Assuming this is not already a part of our release procedure, +1.

Better yet if we can make it part of the release procedure, I think.